Phonebook/phonebook/values.yaml

```diff
@@ -0,0 +1,82 @@
+# Default values for phonebook.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# General Variables
+host: &host 'demo-phonebook.me'
+contactUrl: 'https://github.com/T-Systems-MMS/phonebook/issues'
+roomPlanningToolUrl: 'undefined'
+contactEmail: &contactEmail 'daniel-habenicht@outlook.de'
```

Comment:
maybe we want create an issue for that and think about what e-mail address we should use for that. maybe we create a function mailbox so we all get information for something that went wrong there.

Reply:
good idea, your turn?

Reply:
```suggestion
contactEmail: &contactEmail 'phonebook-t-systems-mms@mg.telekom.de'
```

Reply:
updated

Reply:
i will also add this to docs later